Examples/ByFeatures/DependencyInjection/DependencySteps.fs

```diff
@@ -0,0 +1,49 @@
+﻿module DependencySteps
```

Comment:
my preference would be to have the file split by step classes - it will than be closer to real use and as I plan to use the same test in functional injection it will be better comparison